lms/djangoapps/branding/views.py

```diff
@@ -256,19 +265,25 @@ def footer(request):
     # Override the language if necessary
     language = request.GET.get('language', translation.get_language())
 
+    lang_opts = request.GET.get('language-selector-opts', '')
```

Comment:
NIT: Can probably shorten the query string to `language-selector`.  If we want to keep options in the name, then we should use the full word.